keyboards/darkproject/kd87b_ltd/readme.md

```diff
@@ -0,0 +1,25 @@
+# Dark Project KD87b LTD
+
+![Dark Project KD87b LTD](https://i.postimg.cc/qBcnzcwM/KD87b-LTD.png)
```

Comment:
image isn't rendering correctly, can you please re-host this image on imgur?

Reply:
I uploaded image to imgur, please check it now.